### ISSUE TITLE
[a11y]:After high contrast with white is activated, the On/Off icon that convey important information are not visible 

### ISSUE THREAD

**Issue by wlinacdl** [carbon-design-system/carbon#11648]
### Package

carbon-components

### Browser

Chrome

### Operating System

Windows

### Package version

Latest version

### Automated testing tool and ruleset

Windows setting to turn on 'High Contrast White' or 'High Contrast Black' Setting

### Description

After high contrast is activated, the On/Off that convey important information are not visible

<img width="645" alt="image" src="https://user-images.githubusercontent.com/16383295/174567720-3143813c-c321-4dc6-b47d-77ef4f1fdae6.png">

### Reproduction/example

NA

### Steps to reproduce

Pre-steps： enable 'high contrast with white' for OS theme
Steps:

Go to resource, for example: 'TestGVT测试'
https://dev.console.test.cloud.ibm.com/dns-svcs/crn%3Av1%3Astaging%3Apublic%3Adns-svcs%3Aglobal%3Aa%2F01652b251c3ae2787110a995d8db0135%3Aa9bdf1e5-439a-4241-aa16-cd2a5dd79a9e%3A%3A?paneId=glb
Click on 'Global load balancer' tab
Go to tab 'Origin Pools'

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
@wlinacdl please provide more detailed information including:

- A public reproduction - [here's a template](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/examples/vite?file=src%2FApp.jsx)
- What component is defective
- Confirmation of what package and version you are using - `carbon-components-react`, `@carbon/react`, etc.

**Comment by jnm2377** [comment]
@tay1orjones We've had a few issues with HCM White in the past: https://github.com/carbon-design-system/carbon/pull/10639#issuecomment-1031969494

I checked our current storybook to see if it's still an issue or if it was fixed, but it looks like it's still an issue. Specifically, in the past, the issue was w/ icons not showing up. Although from @wlinacdl screenshot and my current checking of v11 storybook, it looks like the Toggle component also doesn't work in either Black or White HCM.

Perhaps it's worth auditing all of our components to check if everything is rendering correctly in both HCM Black and HCM White.

#### batch action icons are missing in HCM W
<img width="1285" alt="Screen Shot 2022-06-23 at 3 31 08 PM" src="https://user-images.githubusercontent.com/32556167/175395402-1ec0262c-b6cc-4fa2-934d-420d8f5cf19b.png">

<img width="1274" alt="Screen Shot 2022-06-23 at 3 29 44 PM" src="https://user-images.githubusercontent.com/32556167/175395388-6116f6fc-4b25-4136-b80a-3fd74af5d93c.png">

#### toggle just isn't visible at all in either HCM
<img width="171" alt="Screen Shot 2022-06-23 at 3 31 18 PM" src="https://user-images.githubusercontent.com/32556167/175395405-63200085-27a1-419d-ab49-7b9fea7b6f42.png">

<img width="147" alt="Screen Shot 2022-06-23 at 3 29 58 PM" src="https://user-images.githubusercontent.com/32556167/175395399-ca1e2b5e-7c95-4f8b-afe6-9dceb694fe1b.png">

**Comment by jnm2377** [comment]
rel: https://github.com/carbon-design-system/carbon/issues/10458
rel: https://github.com/carbon-design-system/carbon/issues/9548
rel: https://github.com/carbon-design-system/carbon/issues/11557

**Comment by hnjoshi** [comment]
https://stackblitz.com/edit/github-zpzwwj?file=src/App.jsx

@tay1orjones I have added the code sample but I am unable to resolve the cloud pal dependency. Would it be possible for you to help check on it?

**Comment by tay1orjones** [comment]
@hnjoshi I'm not sure if you can configure cloud pal on an external host since it needs the artifactory/npm authentication. Ideally the repro would contain an isolated example of just the Carbon components not working as you'd expect.

**Comment by hnjoshi** [comment]
@tay1orjones in that case does the code sample in https://stackblitz.com/edit/github-zpzwwj?file=src/App.jsx
helps you validate the issue on your end? I don't know if I can do anything on my end to provide you a working demo.

**Comment by tw15egan** [comment]
FWIW the issue is no longer present in V11 (Windows 11 Desert / White theme), but I'm still seeing it on the latest version of V10. 

![image (8)](https://user-images.githubusercontent.com/11928039/190250706-24f3c8e8-ab99-4e0c-a58d-ce555ede5492.png)

**Comment by SongHaiyan2022** [comment]
@tw15egan  I check in Win 11 and Win 10, I am still seeing it on both system.
![image](https://user-images.githubusercontent.com/113488472/190547175-02bc9660-bfa7-4899-98f6-63b99ce22e84.png)
![image](https://user-images.githubusercontent.com/113488472/190547400-6691836d-eca5-439e-ab46-3e9f6de45519.png)

**Comment by tw15egan** [comment]
Yes, I meant in Carbon V11 it is fixed. I have a PR up to backport the fix to V10
